checkov/arm/checks/resource/AzureDefenderOnSqlServersVMS.py

```diff
@@ -0,0 +1,31 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.arm.base_resource_check import BaseResourceCheck
+from typing import List, Dict, Any
+
+
+class AzureDefenderOnSqlServersVMS(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure that Azure Defender is set to On for SQL servers on machines"
+        id = "CKV_AZURE_79"
+        supported_resources = ("Microsoft.Cache/Redis",)
```

Comment:
Probably should be `Microsoft.Security/pricings` type and not `Microsoft.Cache/Redis`